docs/source/_gallery_original/index.rst

```diff
@@ -23,6 +23,7 @@ Contents
 
    vLLM <frameworks/vllm>
    Hugging Face TGI <frameworks/tgi>
+   Ollama <frameworks/ollama>
```

Comment:
I found it a bit odd to group it under "Inference Engines". How about we rename to "Inference Frameworks"?